shenanigan/trainers/core.py

```diff
@@ -28,6 +28,16 @@ def __init__(self, model, batch_size, save_location,
         self.minimum_val_loss = None
         self.show_progress_bar = show_progress_bar
         self.callbacks = callbacks if callbacks is not None else []
+        self.continue_training = continue_training
+
+        self.ckpt = tf.train.Checkpoint(
+            step=tf.Variable(1),
```

Comment:
Would this not start at zero? It looks like you increment before saving

Reply:
good spot, made the update